nix-darwin/default.nix

```diff
@@ -8,7 +8,7 @@ in rec {
     ghc = pkgs_mac.haskellPackages.ghcWithPackages (ps: [ ps.turtle ps.universum ps.megaparsec ]);
   in pkgs_mac.stdenv.mkDerivation {
     name = "deploy-nix-darwin";
-    buildInputs = [ ghc pkgs.nixStable ];
+    buildInputs = [ ghc ];
     shellHook = "eval $(egrep ^export ${ghc}/bin/ghc)";
```

Comment:
and if its not useful as a shell expression, maybe also remove the `shellHook`?

Reply:
@rvl said it's still useful, so left it in.